From e0ca91175caab9031321fa441f440fdc3ad3d4f5 Mon Sep 17 00:00:00 2001
From: Richard Hu <richard.hu@technexion.com>
Date: Tue, 4 Aug 2015 11:10:32 +0800
Subject: [PATCH] edm-cf-imx6: put zImage and dtb into root directory instead
 of /boot in 1st FAT partition to follow the rule in
 image_types_fsl.bbcleass in yocto.

---
 board/technexion/edm-cf-imx6/edm-cf-imx6.c |    8 ++++----
 include/configs/edm-cf-imx6.h              |    2 +-
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/board/technexion/edm-cf-imx6/edm-cf-imx6.c b/board/technexion/edm-cf-imx6/edm-cf-imx6.c
index 536adb1..2c2f8f0 100644
--- a/board/technexion/edm-cf-imx6/edm-cf-imx6.c
+++ b/board/technexion/edm-cf-imx6/edm-cf-imx6.c
@@ -596,16 +596,16 @@ int misc_init_r(void)
 			case EDM1_CF_IMX6_SOM:
 			case EDM2_CF_IMX6_SOM:
 				if (is_cpu_type(MXC_CPU_MX6Q) || is_cpu_type(MXC_CPU_MX6D))
-					setenv("fdt_file", "boot/imx6q-edm1-cf.dtb");
+					setenv("fdt_file", "imx6q-edm1-cf.dtb");
 				else
-				setenv("fdt_file", "boot/imx6dl-edm1-cf.dtb");
+					setenv("fdt_file", "imx6dl-edm1-cf.dtb");
 				break;
 			case WANDBOARD_B1_SOM:
 			case WANDBOARD_C1_SOM:
 				if (is_cpu_type(MXC_CPU_MX6Q) || is_cpu_type(MXC_CPU_MX6D))
-					setenv("fdt_file", "boot/imx6q-wandboard.dtb");
+					setenv("fdt_file", "imx6q-wandboard.dtb");
 				else
-					setenv("fdt_file", "boot/imx6dl-wandboard.dtb");
+					setenv("fdt_file", "imx6dl-wandboard.dtb");
 				break;
 			}
 		}
diff --git a/include/configs/edm-cf-imx6.h b/include/configs/edm-cf-imx6.h
index 4a60e79..85b9ba0 100644
--- a/include/configs/edm-cf-imx6.h
+++ b/include/configs/edm-cf-imx6.h
@@ -142,7 +142,7 @@
 #ifdef CONFIG_DEBUG_CONSOLE
 #define CONFIG_EXTRA_ENV_SETTINGS \
 "script=boot.scr\0" \
-"image=boot/zImage\0" \
+"image=zImage\0" \
 "console=ttymxc0\0" \
 "splashpos=m,m\0" \
 "fdt_high=0xffffffff\0" \
-- 
1.7.9.5

